update_python_packages.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import subprocess
+import os
+import sys
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
```

Comment:
Can we just import the module instead and then use the module name as a prefix to request sub items? This way, we know where the item is coming from.

```suggestion
import pathlib
import tempfile
import zipfile
```

Then, we want to alpha order sort all those imports